cep-7.md

```diff
@@ -0,0 +1,35 @@
+<table>
+<tr><td> Title </td><td> Conda Capitalization Standard </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Katherine Kinnaman &lt;kkinnaman@anaconda.com&gt;</td></tr>
+<tr><td> Created </td><td> Jun 28, 2022</td></tr>
+<tr><td> Updated </td><td> Jun 28, 2022</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+The capitalization conventions for conda need to be decided and written down within the conda docs so that they can be enforced during PRs, for consistency within written materials about and related to conda.
+
+## Specification
+
+This change will only affect the text of the conda docs, so shouldn't need any special technical considerations. 
+
+## Motivation
+
+The capitalization of C/conda can be inconsistent. It should be formally decided and written down so that it can be enforced for consistency in any documentation or written materials about conda.
+
+## Alternatives
+
+- Conda should be lowercase except when used at the beginning of a sentence. When speaking about the command, code formatting should be used (i.e. `conda`)
+
+- Conda should be capitalized when speaking about the organization/product and lowercase, with code formatting, when discussing the command
```

Comment:
Move the choice you are advocating for up to the "Specification".

"Alternatives" should indicate other options that were considered but rejected. Include any reasoning for why alternative ideas were rejected.